src/app/common/api/wrapped-fetch.ts

```diff
@@ -1,4 +1,5 @@
 const hiroHeaders: HeadersInit = {
+  'x-api-key': process.env.X_API_KEY || '',
```

Comment:
```suggestion
  'x-api-key': process.env.X_API_KEY ?? '',
```

Suggest favouring `??` operator in cases like this

Reply:
Tbh, I just wasn't aware of our current setup. I haven't used a new environment variable we need to keep private. Typically, I've used an `.env` file, so that is what I did and realized the variable wasn't being recognized. Anyway, I am happy to switch to our current setup for private variables. I'll look into it today.